.changeset/fifty-hotels-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: remove `postcss-media-query-parser` dependency
```

Comment:
minor

You can use `Removed:`.
cf https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md#1500

Reply:
thank you 🙇